mythril/laser/ethereum/instructions.py

```diff
@@ -68,14 +69,15 @@ class StateTransition(object):
     if `increment_pc=True`.
     """
 
-    def __init__(self, increment_pc=True, enable_gas=True):
+    def __init__(self, increment_pc=True, enable_gas=True, static_check=False):
```

Comment:
Can you change the variable name to something that is more clear about the purpose.